src/xclim/indices/_threshold.py

```diff
@@ -3633,3 +3638,148 @@ def wet_spell_max_length(
         resample_before_rl=resample_before_rl,
         **indexer,
     )
+
+
+@declare_units(
+    snd="[length]",
+    prsn="[precipitation]",
+    snd_thresh="[length]",
+    prsn_thresh="[length]",
+)
+def holiday_snow_days(
+    snd: xarray.DataArray,
+    snd_thresh: Quantified = "20 mm",
+    op: str = ">=",
+    date_start: str = "12-25",
+    date_end: str | None = None,
+    freq: str = "YS",
+) -> xarray.DataArray:  # numpydoc ignore=SS05
+    """
+    Christmas Days.
+
+    Whether there is a significant amount of snow on the ground on December 25th (or a given date range).
+
+    Parameters
+    ----------
+    snd : xarray.DataArray
+        Surface snow depth.
+    snd_thresh : Quantified
+        Threshold snow amount. Default: 20 mm.
+    op : {">", "gt", ">=", "ge"}
+        Comparison operation. Default: ">=".
+    date_start : str
+        Beginning of analysis period. Default: "12-25" (December 25th).
+    date_end : str, optional
+        End of analysis period. If not provided, `date_start` is used.
+        Default: None.
+    freq : str
+        Resampling frequency. Default: "YS".
+        The default value is chosen for the northern hemisphere.
+
+    Returns
+    -------
+    xarray.DataArray, [bool]
+        Boolean array of years with Christmas Days.
+
+    References
+    ----------
+    https://www.canada.ca/en/environment-climate-change/services/weather-general-tools-resources/historical-christmas-snowfall-data.html
+    """
+    snd_constrained = select_time(
+        snd,
+        drop=True,
+        date_bounds=(date_start, date_start if date_end is None else date_end),
+    )
+
+    xmas_days = count_occurrences(
+        snd_constrained, snd_thresh, freq, op, constrain=[">=", ">"]
+    )
+
+    xmas_days = xmas_days.assign_attrs({"units": "days"})
+    return xmas_days
+
+
+@declare_units(
+    snd="[length]",
+    prsn="[precipitation]",
+    snd_thresh="[length]",
+    prsn_thresh="[length]",
+)
+def holiday_snow_and_snowfall_days(
+    snd: xarray.DataArray,
+    prsn: xarray.DataArray | None = None,
+    snd_thresh: Quantified = "20 mm",
+    prsn_thresh: Quantified = "1 cm",
+    snd_op: str = ">=",
+    prsn_op: str = ">=",
+    date_start: str = "12-25",
+    date_end: str | None = None,
+    freq: str = "YS-JUL",
+) -> xarray.DataArray:
+    r"""
+    Perfect Christmas Days.
+
+    Whether there is a significant amount of snow on the ground and measurable snowfall occurring on December 25th.
+
+    Parameters
+    ----------
+    snd : xarray.DataArray
+        Surface snow depth.
+    prsn : xarray.DataArray
+        Snowfall flux.
+    snd_thresh : Quantified
+        Threshold snow amount. Default: 20 mm.
+    prsn_thresh : Quantified
+        Threshold snowfall flux. Default: 1 mm.
```

Comment:
```suggestion
        Threshold daily snowfall liquid-water equivalent thickness. Default: 1 mm.
```
1 mm is not a flux, it is a liquid-water equivalent thickness!

Reply:
You'll note that this means the  `snd_thresh` is in actual thickness and `prsn_thresh` is in liquid-water equivalent thickness. Welcome to the world of snow.

Reply:
Thanks, I hate it.

For the `prsn` indicator, ECCC mentions the term "measurable snowfall", which gives me the impression that it needs to be enough to not be considered a trace amount.

If we look at [January 2024 at YUL](https://climate.weather.gc.ca/climate_data/daily_data_e.html?timeframe=2&hlyRange=2013-02-13%7C2025-01-09&dlyRange=2013-02-14%7C2025-01-09&mlyRange=%7C&StationID=51157&Prov=QC&urlExtension=_e.html&searchType=stnName&optLimit=yearRange&StartYear=1840&EndYear=2025&selRowPerPage=25&Line=4&searchMethod=contains&txtStationName=montreal&time=LST&Day=9&Year=2024&Month=1#) or other months, the smallest increment above 0.0 cm seems to be 0.4 cm of snow (not swe).

We have the `snd_to_snw` conversion for estimating the amount based on the density of snow. Would it make sense to use taht here?

Reply:
I know that this is quite complex for a Friday afternoon indicator, but What if we were to convert as follows:
```python
snw_equivalent = convert_units_to(prsn, "kg m-2")
snd_from_falling_snow = snw_to_snd(snw_equivalent)
cond = snd_from_falling_snow > prsn_thresh
```

It would probably make sense to rename `prsn_thresh` in this case...

Reply:
I would argue against using `snw_to_snd` here has to not introduce the snow density uncertainty, better ask for a liquid water equivalent and let the user to it's own (mental) conversion ?